content/docs/examples/multicluster/icp/index.md

```diff
@@ -146,17 +150,18 @@ This example uses `cluster-1` as the local Istio control plane and `cluster-2` a
 
 The following example enables [automatic sidecar injection](/docs/setup/kubernetes/additional-setup/sidecar-injection/#automatic-sidecar-injection).
 
-1.  Install `bookinfo` on the first cluster `cluster-1`. Remove `reviews-v3` deployment to deploy on remote:
+1.  Install `bookinfo` on the first cluster `cluster-1`. Remove `reviews-v3` deployment to deploy on remote cluster `cluster-2`:
```

Comment:
```suggestion
1.  Install `bookinfo` on the first cluster `cluster-1`. Remove the `reviews-v3` deployment which will be deployed on cluster `cluster-2` in the following step:
```

Reply:
```suggestion
1.  Install the `bookinfo` app on the `cluster-1` cluster. Remove the `reviews-v3` service since the next step is to deploy it on the `cluster-2` cluster:
```